### PR TITLE
Spanish localization update for v3.2

### PR DESCRIPTION
**Pull request by LuisPignataro**
I have updated the localization files in Spanish with the changes that were made in English for version 3.2

### CODE REVIEW

File: framework/src/Volo.Abp.Ddd.Application.Contracts/Volo.Abp.Ddd.Application.Contracts.csproj [hunk 1]
```diff
@@ -19,6 +19,10 @@
     <Content Remove="Volo\Abp\Application\Localization\Resources\AbpDdd\*.json" />
   </ItemGroup>
 
+  <ItemGroup>
```
Comment (hikalkan) [inline framework/src/Volo.Abp.Ddd.Application.Contracts/Volo.Abp.Ddd.Application.Contracts.csproj]: Please remove this ItemGroup (it is probably auto added by the VS).

File: framework/src/Volo.Abp.Timing/Volo.Abp.Timing.csproj [hunk 2]
```diff
@@ -19,6 +19,10 @@
     <Content Remove="Volo\Abp\Timing\Localization\**\*.json" />
   </ItemGroup>
 
+  <ItemGroup>
```
Comment (hikalkan) [inline framework/src/Volo.Abp.Timing/Volo.Abp.Timing.csproj]: Please remove this ItemGroup (it is probably auto added by the VS).

File: framework/test/Volo.Abp.AspNetCore.Mvc.Tests/Volo.Abp.AspNetCore.Mvc.Tests.csproj [hunk 3]
```diff
@@ -19,6 +19,10 @@
     <EmbeddedResource Include="Volo\Abp\AspNetCore\Mvc\Localization\Resource\*.json" />
   </ItemGroup>
 
+  <ItemGroup>
```
Comment (hikalkan) [inline framework/test/Volo.Abp.AspNetCore.Mvc.Tests/Volo.Abp.AspNetCore.Mvc.Tests.csproj]: Please remove this ItemGroup (it is probably auto added by the VS).

**Review by hikalkan — CHANGES_REQUESTED** [review]

**Comment by hikalkan** [comment]
Thanks a lot. I have a few small change requests.

**Comment by LuisPignataro** [comment]
Ready, clean up the projects!

**Comment by hikalkan** [comment]
Thanks :)

**Pull request merged**